libs/fetcher.js

```diff
@@ -49,9 +48,7 @@ function parseRequest(req) {
 }
 
 function sanitizeResourceName(resource) {
-    return resource
-        ? resource.replace(RESOURCE_SANTIZER_REGEXP, '*')
-        : resource;
+    return resource ? resource.replace(/[^\w.]+/g, '*') : resource;
```

Comment:
Was this to avoid a bug? We usually keep the regex out to avoid recreating the regex object each request

Reply:
Oh, I didn't know that. I'll remove this commit.